deployment/test/rules.jrl

```diff
@@ -0,0 +1,16 @@
+p({
+  "class": "foam.nanos.dig.DUGRule",
+  "id": "foam-nanos-dig-DUGLoopbackBucketTest",
```

Comment:
I didn't see pom.js in this directory but the dug rule is loaded on my local build `./build.sh -Japi,smte,scotiabank -dj`, how does it work? did I miss something?

Reply:
deployment/test should only be loaded with -Jtest or -t  or -T 
I'll 'not' test locally!

Reply:
It would appear the build is broken and all journals from foam3/deployment are included